Client/game_sa/CModelInfoSA.h

```diff
@@ -290,7 +302,18 @@ class CVehicleModelInfoSAInterface : public CBaseModelInfoSAInterface
     float                               fSteeringAngle;
     CVehicleModelVisualInfoSAInterface* pVisualInfo;            // +92
     char                                pad3[464];
-    char                                pDirtMaterial[64];            // *RwMaterial
+
+    union
+    {
+        struct
+        {
+            RpMaterial** m_dirtMaterials;
+            size_t       m_numDirtMaterials;
+            RpMaterial*  m_staticDirtMaterials[30];
+        };
+        RpMaterial* pDirtMaterial[32];
+    };
+
```

Comment:
This struct has a different size now. 
Old size: 64 bytes
New size: 128 bytes

Did you break the next data offsets?

Reply:
Yes. None of the following fields are used anywhere, but it still shouldn’t be like this. What do you suggest

Reply:
Can you fill the interface with right info?

Reply:
Yes, but structure also affects class size. How to solve this? Fields needed include numMaterials, staticMaterials etc

Reply:
Remove `char pad4[64];`